audisp-tacplus.c

```diff
@@ -600,16 +498,18 @@ static void get_acct_record(auparse_state_t *au, int type)
         logptr += llen;
         tlen += llen;
     }
+    
+    /* try remove user secret */
+    if (remove_user_secret(logbase, fixed_log_buffer, sizeof(fixed_log_buffer)) == USER_SECRET_FIXED) {
+        logbase = fixed_log_buffer;

```

Comment:
>logbase [](http://example.com/codeflow?start=8&length=7)

It is not reader-friendly to change variable used in the condition check. How about assign `remove_user_secret()` result to a variable, and check that variable before calling following `send_tacacs_acct()`?

Reply:
Fixed.

Reply:
Still changing `logbase` which is used by condition check. However this is minor issue.

Reply:
Fixed